src/calculateRentalCost.js

```diff
@@ -26,9 +26,15 @@
  *
  * @return {number}
  */
+const rentcost = 40;
+const discountMoreWeek = 50;
+const discountLessweek = 20;
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = rentcost * days;
+
+  return days < 3 ? result : days < 7
+    ? result - discountLessweek : result - discountMoreWeek;
```

Comment:
Don't nest the ternary operator inside another ternary operator. It is hard to read